src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -443,10 +452,19 @@ namespace AppInstaller::CLI::Workflow
         }
 
         SearchRequest searchRequest;
+
         if (args.Contains(Execution::Args::Type::Query))
         {
             searchRequest.Query.emplace(RequestMatch(matchType, args.GetArg(Execution::Args::Type::Query)));
         }
+        if (args.Contains(Execution::Args::Type::MultiQuery))
+        {
+            auto queries = args.GetArgs(Execution::Args::Type::MultiQuery);
+            for (const auto& query : *queries)
+            {
+                searchRequest.Query.emplace(RequestMatch(matchType, query));

```

Comment:
I don't know why I assumed it was a vector... This was only for the list command. I don't think it is that important to have multi query in that one, so I just reverted that one back to single query instead of reworking it..